rero_mef/cli.py

```diff
@@ -514,58 +500,57 @@ def ordert_data(data):
         click.echo(f'Changed file: {file_name_diff}')
         click.echo(f'Deleted file: {file_name_delete}')
 
-    compaire_data = SqliteDict(sqlite_dict, autocommit=True)
-    if csv_metadata_file_compair and not entity:
+    compair_data = SqliteDict(sqlite_dict, autocommit=True)
```

Comment:
Why not using redis?

Reply:
Because I can use it like a dictionary with less memory consumption.